### PR TITLE
Generate code

### PR DESCRIPTION
**Pull request by thiagovsk**

### CODE REVIEW

File: source/code_table.c [hunk 1]
```diff
@@ -105,3 +106,24 @@ line *find_line(line *head_code_table, int number)
         return NULL;
 }
 
+int write_code_table(line *head_code_table){
```
Comment (lucaskanashiro) [inline source/code_table.c]: open statement ('{') in line below

File: source/code_table.c [hunk 2]
```diff
@@ -105,3 +106,24 @@ line *find_line(line *head_code_table, int number)
...
+
+    mkdir("output/", 0700);
+    FILE *file = fopen("output/safe_code.c","w+");;
```
Comment (lucaskanashiro) [inline source/code_table.c]: remove the second dot comma (';')

File: tests/source/test_code_table.c [hunk 3]
```diff
@@ -103,6 +105,50 @@ void test_find_symbol(void) {
     CU_ASSERT_EQUAL(element_4, NULL);
 }
 
+void test_write_code(void) {
```
Comment (lucaskanashiro) [inline tests/source/test_code_table.c]: open statemant in ine below ('{')

File: tests/source/test_code_table.c [hunk 4]
```diff
@@ -103,6 +105,50 @@ void test_find_symbol(void) {
...
+    code_table = create_code_table();
+
+    insert_line(code_table, "test0", 1);
+    insert_line(code_table, "test1", 2);
+    insert_line(code_table, "test2", 3);
+
+    CU_ASSERT_EQUAL(write_code_table(code_table),1);
+}
+
+void test_write_code_without_table(void){
```
Comment (lucaskanashiro) [inline tests/source/test_code_table.c]: open statemant in ine below ('{')

File: tests/source/test_code_table.c [hunk 5]
```diff
@@ -103,6 +105,50 @@ void test_find_symbol(void) {
...
+  insert_line(code_table, "test0", 1);
+  insert_line(code_table, "test1", 2);
+  insert_line(code_table, "test2", 3);
+
+  CU_ASSERT_EQUAL(write_code_table(code_table),1);
+}
+
+void test_write_code_table_sequential(void) {
```
Comment (lucaskanashiro) [inline tests/source/test_code_table.c]: open statemant in ine below ('{')

File: tests/source/test_code_table.c [hunk 6]
```diff
@@ -103,6 +105,50 @@ void test_find_symbol(void) {
...
+
+    insert_line(code_table, "test0", 1);
+    insert_line(code_table, "test1", 2);
+    insert_line(code_table, "test2", 3);
+
+    write_code_table(code_table);
+
+    FILE *file = fopen("output/safe_code.c","r+");
+
+    while (fgets (line_content, 200, file) != NULL) {
```
Comment (lucaskanashiro) [inline tests/source/test_code_table.c:149]: open statemant in ine below ('{')

File: source/code_table.c [hunk 7]
```diff
@@ -105,3 +106,24 @@ line *find_line(line *head_code_table, int number)
...
+
+    if(code_table_is_empty(head_code_table))
+        return 0;
+
+    line *iterator = (line *) malloc(sizeof(line));
+    iterator = head_code_table->next;
+    fprintf(file, "//Safe-C Output\n//Now you code is Safe!\n");
+
+    while (iterator != NULL) {
```
Comment (lucaskanashiro) [inline source/code_table.c]: open statemant in ine below ('{')

**Comment by thiagovsk** [comment]
@lucaskanashiro done.

#tpp

**Pull request merged**